packages/shopify-app-remix/src/server/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  rules: {
+    'no-restricted-imports': [
+      'error',
+      {
+        patterns: [
+          {
+            group: ['../react/*'],
+            message:
+              'Importing from src/react in src/server is not allowed as they must be kept separate',
```

Comment:
Worthwhile explaining why? E.g: It will blow up Remix due to different runtimes.